microprofile/telemetry/src/test/java/io/helidon/microprofile/telemetry/TelemetryTest.java

```diff
@@ -0,0 +1,29 @@
+package io.helidon.microprofile.telemetry;
+
+import io.helidon.microprofile.tests.junit5.AddBean;
+import io.helidon.microprofile.tests.junit5.HelidonTest;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.client.WebTarget;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+@HelidonTest
+@AddBean(TestResource.class)
+class TelemetryTest {
```

Comment:
Again, this test has nothing to do with telemetry, it just validates you get a response

Reply:
fixed